### PR TITLE
Added Material Light: a library of UI components

### PR DESCRIPTION
**Pull request by YagoLopez**
There are several alternatives into the Material Design field for Angular 2 but this one has added value and advantages respect others in my opinion (Take a loot at the Readme.md for details). There is also an online demo to see it in action. (API documentation is still lacking but in process).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,13 +169,15 @@ Table of contents:
 #### Server-Side Rendering
 * [Angular 2 Universal Repository (GitHub)](https://github.com/angular/universal)
 
-#### Material Design
+####
```
Comment (rolandjitsu) [inline README.md]: This must have been done accidentally. Could you undo please?

Comment (YagoLopez) [inline README.md]: Sorry if I caused problems. Do you mean I delete all the text I added and restore to the previous state or just arranging the lines 172 and 173 could the pull request be accepted? I would like to collaborate the best way as possible.

Comment (rolandjitsu) [inline README.md]: Yes, the headline read `#### Material Design`, now it's just `Design`. It should be as it was before it was changed, as in `#### Material Design`. Thanks.

**Comment by YagoLopez** [comment]
Excuse me for bothering you again but Travis seems to give an error about url validation, Dont know what excatly is happening.

**Comment by rolandjitsu** [comment]
Don't worry about that, I'll fix those later.

**Comment by YagoLopez** [comment]
Alright thank you for all :) 👍

**Pull request merged**